website/content/docs/rfcs/namespace-sealing.mdx

```diff
@@ -116,18 +116,15 @@ A3[Share 3] -->|Recovers| B[Namespace Shamir's Key Encryption Key]
     RBE -->|Encrypts| ENSRK[Encrypted Namespace Root Key]
 ```
 
-With Per-Namespace Sealing, the responsibility of Root Barrier Keyring will be extended. Root Barrier Keyring will be used to encrypt the Shamirs Key Encryption Key (KEK) for a Namespace. This has the implication that a Namespace Shamirs KEK first needs to be decrypted (Global needs to be unsealed) before a namespace can be unsealed.
+With Per-Namespace Sealing, the responsibility of Root Barrier Keyring will be extended. The parent namespace's Barrier Keyring will be used to encrypt the sealed namespace's stored Shamirs Key Encryption Key (KEK). This has the implication that a Namespace Shamirs KEK first needs to be decrypted (implying that the parent or root namespace needs to be unsealed) before a namespace can be unsealed.
 
 ### Unsealing Process
 
-1. OpenBao will attempt an unblocking unseal of each sealed namespace (all namespaces with sealing config), skipping Shamir seals
-   - on success: namespace will work as expected and requests can be answered
-   - on fail: namespace remains sealed; returns applicable errors; status can be retrieved from namespace-status endpoint; periodic retry (as HSM can be temporarily unavailable etc.)
-
 1. OpenBao global unseal is successful
 2. OpenBao will attempt an unblocking unseal of each sealed namespace (all namespaces with sealing config), skipping Shamir seals
    - on success: namespace will work as expected and requests can be answered
-   - on fail: namespace remains sealed; returns applicable errors; status can be retrieved from namespace-status endpoint; periodic retry (as HSM can be temporarily unavailable etc.)3. For namespaces that could not be auto-unsealed (and do not have a sealing config), OpenBao allows manual shamir unseal via the respective namespace specific api endpoint.  
+   - on fail: namespace remains sealed; returns applicable errors; status can be retrieved from namespace-status endpoint; periodic retry (as HSM can be temporarily unavailable etc.)
+3. For namespaces that could not be auto-unsealed (and do not have a sealing config), OpenBao allows manual shamir unseal via the respective namespace specific api endpoint.  
```

Comment:
Not sure what that means:

> For namespaces that could not be auto-unsealed (and do not have a sealing config)

If a namespace does not have a sealing config, it is not a sealable namespace